torch/csrc/distributed/c10d/init.cpp

```diff
@@ -136,7 +137,7 @@ PyObject* c10d_init(PyObject* _unused) {
       py::arg("state"),
       py::arg("comm_hook"));
 
-  shared_ptr_class_<::c10d::GradBucket>(module, "GradBucket")
+  shared_ptr_class_<::c10d::GradBucket>(module, "_GradBucket")
       .def(py::init<std::vector<Tensor>&>(), py::arg("tensors"))
       .def(
           "get_tensors",
```

Comment:
Can we add documentation to this method (similar to the documentation in the C++ getTensors method) explaining the replicas?